app/graphql/types/objects/consultation/base.rb

```diff
@@ -40,7 +40,7 @@ def visible?(context)
 				field :responses_reading_times,						Integer, "Reading times of all the responses in this consultation", null: false
 				field :reading_time,											Integer, "Reading time of this consultation summary", null: false
 				field :visibility,												Types::Enums::ConsultationVisibilityType, nil, null: false
-
+				field :questions,													Types::Connections::Question, nil, null: true
```

Comment:
@BALAJI-BALA - I suggest having this as an array rather than connection. 

Connection is generally implemented for pagination